File: Kattis/CPlusPlus/DiceCup.cpp
#include <iostream>

int main() {
	int i, firstRoll, secondRoll, lowestRoll, highestRoll;
	std::cin >> firstRoll >> secondRoll;
	lowestRoll = firstRoll;
	highestRoll = secondRoll;

	if (secondRoll < firstRoll) {
		lowestRoll = secondRoll;
		highestRoll = firstRoll;
	}

	for (i = (lowestRoll + 1); i <= (highestRoll + 1); i++) {
		std::cout << i << std::endl;
	}

	std::cin.get();
	std::cin.ignore();
	return 0;
}

File: Kattis/CSharp/DiceCup.cs
using System;

namespace DiceCup
{
    class DiceCup
    {
        static void Main(string[] args)
        {
            int i, firstRoll, secondRoll, lowestRoll, highestRoll;
            firstRoll = Convert.ToInt32(Console.ReadLine());
            secondRoll = Convert.ToInt32(Console.ReadLine());
            lowestRoll = firstRoll;
            highestRoll = secondRoll;

            if (secondRoll < firstRoll)
            {
                lowestRoll = secondRoll;
                highestRoll = firstRoll;
            }

            for (i = (lowestRoll + 1); i <= (highestRoll + 1); i++)
            {
                Console.WriteLine(i);
            }
        }
    }
}
